prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -92,6 +92,11 @@ spec:
               cpu: 10m
           securityContext:
             privileged: true
+          env:
+          - name: MY_NODE_NAME
```

Comment:
`NODE_NAME` sounds good to me 👍 
`HOSTNAME` could indeed be a bit more obscure, e.g. what would happen if not in the host namespace? (even though this should always be the case for Weave Net)